62.CAD/common.hlsl

```diff
@@ -148,9 +131,65 @@ struct PSInput
     void setBezierP0(float2 p0) { data2.xy = p0; }
     void setBezierP1(float2 p1) { data2.zw = p1; }
     
-    // data3 (zw reserved for later)
+    // data3 xy
     float2 getBezierP2() { return data3.xy; }
     void setBezierP2(float2 p2) { data3.xy = p2; }
+
+    // Curves are split in the vertex shader based on their tmin and tmax
+    // Min curve is smaller in the minor coordinate (e.g. in the default of y top to bottom sweep,
+    // curveMin = smaller x / left, curveMax = bigger x / right)
+    // TODO: possible optimization: passing precomputed values for solving the quadratic equation instead
+
+    // data2, data3, data4
+    float2 getCurveMinA() { return data2.xy; }
+    float2 getCurveMinB() { return data2.zw; }
+    float2 getCurveMinC() { return data3.xy; }
```

Comment:
See how in @Przemog1 PR he constructs and decomposits structs in set/gets here. you don't have to always return basic types, or get basic types inputs here.
after that merge you'll be setting and getting a "nbl::hlsl::shapes::Quadratic" struct which has the A, B, C values inside. instead of doing get and set for A,B,C seperately

but good stuff that you looked out for possibilities of reuse